### ISSUE TITLE
cmake 3.5.2 configuration fails

### ISSUE THREAD

**Issue by in3otd** [radareorg/cutter#1796]
(continued from [here](https://github.com/radareorg/cutter/pull/1553#issuecomment-533916719), text copied below)

**Environment information**
* Operating System: Slackware 14.2
* Cutter version: pulled latest from repo (a15d104b4559310cfe9603f6f979496656c1adf9)

**Describe the bug**
`cmake` 3.5.2 fails even if the docs say that >3.1 should be ok

**To Reproduce**
Try to build with a `cmake` < 3.7 :grin:

I have `cmake` version 3.5.2 and it fails since it does not know about `VERSION_GREATER_EQUAL` - apparently that was introduced in version 3.7.
Google told me that I could change the line
```
if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")
```
in `CMakeMakeLists.txt` with
```
if (POLICY CMP0074)
```
With this change `cmake` goes a little further, then fails again but I think for some unrelated reason.

**Comment by karliss** [comment]
Progress update:
* There is one more cmake version check
* UpdateWorker depends on QVersionNumber, simplest thing is to just disable.

**Comment by karliss** [comment]
It still needs some cleanup but got it compiling and working on Ubuntu 16.04 with default packages - QT 5.5  and Cmake 3.5.

**Comment by karliss** [comment]
@in3otd Can you check the PR branch if it works on your system. If you are using CUTTER_USE_BUNDLED_RADARE2 you will still need a sufficiently new meson to build r2, it be easily installed using pip. I also have no idea if any of the Cutter Pythons stuff works so better start without setting any of the options for enabling python.

**Comment by in3otd** [comment]
thanks, #1805 works fine on my system when using `CUTTER_USE_BUNDLED_RADARE2`. I was able to build and run Cutter.
I have some warnings from `cmake` but I guess those are "normal":

```
$ cmake ..
CMake Warning at CMakeLists.txt:91 (find_package):
  By not providing "FindKF5SyntaxHighlighting.cmake" in CMAKE_MODULE_PATH
  this project has asked CMake to find a package configuration file provided
  by "KF5SyntaxHighlighting", but CMake did not find one.

  Could not find a package configuration file provided by
  "KF5SyntaxHighlighting" with any of the following names:

    KF5SyntaxHighlightingConfig.cmake
    kf5syntaxhighlighting-config.cmake

  Add the installation prefix of "KF5SyntaxHighlighting" to CMAKE_PREFIX_PATH
  or set "KF5SyntaxHighlighting_DIR" to a directory containing one of the
  above files.  If "KF5SyntaxHighlighting" provides a separate development
  package or SDK, be sure it has been installed.


-- Checking for module 'libgvc'
--   No package 'libgvc' found
-- 
-- Building Cutter version 1.9.0
-- Options:
-- - Bundled radare2: ON
-- - Python: OFF
-- - Python Bindings: OFF
-- - Crash Handling: OFF
-- - KSyntaxHighlighting: OFF (KSyntaxHighlighting not found)
-- - Graphviz: OFF
-- 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/cutter/src/cmake_build
```

I've tried also without `CUTTER_USE_BUNDLED_RADARE2` since I have also `r2` previously installed but for some reason `cmake` does not find it, or does not find some related library/module.
The error message looks a bit cryptic to me:

```
-- Checking for one of the modules 'IMPORTED_TARGET;r_core'
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find Radare2 (missing: Radare2_TARGET)
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindRadare2.cmake:97 (find_package_handle_standard_args)
  CMakeLists.txt:57 (find_package)


-- Configuring incomplete, errors occurred!
```

is this related to the `r_core` library? Because that seems to be present:
```
$ pkg-config --modversion r_core
3.9.0
```

**Comment by karliss** [comment]
That's a problem with cmake version and the way r2 gets added to Cutter project. IMPORTED_TARGET option was implemented in cmake 3.6.
